### PR TITLE
Task 031,Task 027, Task 028, Task 029, Task 030, and Task 033

### PR DESCRIPTION
**Pull request by mbagwell**
This commit gives the illusion that time is passing by making the sky and clouds fade and reappear over time.

### CODE REVIEW

**Review by WilltheWys — COMMENTED** [review]
Adding these changes makes the background look a little more pleasing. Making the Background Dynamic and changing is more pleasing to the eyes. It makes everything look like its moving.

**Review by NotHacker101 — APPROVED** [review]
I like that you explain what you did in a short way !!! keep up the good work.

**Comment by DSF256** [comment]
This looks pretty cool; a day and night transition makes the game feel a lot more lively.  I would probably recommend making the day/night cycle transitions somewhat longer, though.  The sky changes every 1 second, which makes days go by a bit too quickly.  I would probably change it to once every 5 or 6 seconds.

**Review by DSF256 — CHANGES_REQUESTED** [review]
Okay, I think I screwed that last one up.  Anyway, I think this is good save for the interval between sky changes.  I tested a 5-second interval in my own project, and it seemed to feel more natural.  Hope this helps.

File: 2d/platformer/src/Level/Sky.gd [hunk 1]
```diff
@@ -0,0 +1,30 @@
+extends ParallaxLayer
+
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+onready var sprite = $Sprite
```
Comment (reid-wynkoop) [inline 2d/platformer/src/Level/Sky.gd:7]: does sprite ever get used?

**Review by reid-wynkoop — APPROVED** [review]

File: 2d/platformer/src/Level/Sky.gd (hunk 1 above)
Comment (mbagwell) [reply 2d/platformer/src/Level/Sky.gd:7]: No.

**Pull request merged**